packages/requisitions/src/ResponseRequisition/DetailView/ResponseLineEdit/ResponseLineEdit.tsx

```diff
@@ -22,16 +23,29 @@ export const ResponseLineEdit = ({
   onClose,
   line,
 }: ResponseLineEditProps) => {
-  const isDisabled = useIsResponseRequisitionDisabled();
+  const [currentLine, setCurrentLine] = useBufferState(line);
+  const disabled = useIsResponseRequisitionDisabled();
   const { Modal } = useDialog({ onClose, isOpen });
-  const { draft, isLoading, save, update } = useDraftRequisitionLine(line);
+  const { draft, isLoading, save, update } =
+    useDraftRequisitionLine(currentLine);
+  const { next, isDisabled } = useNextResponseLine(currentLine);
```

Comment:
I think better to keep the `isDisabled` on the form and rename this one to make it clear that it is for the next button only